py/__init__.py

```diff
@@ -337,11 +339,16 @@ def _send(self, msg, endpoint='events', quiet=False):
             )
             return r.text
         except BaseException:
-            if not quiet:
-                print("Exception in user code:")
-                print('-' * 60)
-                traceback.print_exc()
-            return False
+            if self.raise_exceptions:
+                raise ConnectionError("Error connecting to Visdom server")
+            else:
```

Comment:
Ah this is really close - can we instead have a 3 case split, where in this else if self.raise_exceptions is None, you warn that the default is going to change and that users should set `raise_expectations` to `False` to retain their desired behavior?

Reply:
Oh I'm just seeing this. I can get it done by later tonight if that's fine.

Reply:
That also works, no rush on it really. I'm just trying to clean up our PRs before I get #281 in next week so that I prevent annoying merge conflicts. If it's not done then I'll just do it 👍

Reply:
Alright, I've added the recommendation. @JackUrb let me know if this looks good.